src/events/request/RequestEventHandler.ts

```diff
@@ -91,6 +124,16 @@ export default class RequestEventHandler implements EventHandler<'message'> {
 		}
 	};
 
+	private getTickets( content: string ): string[] {
+		let ticketMatch: RegExpExecArray;
+		const regex = MentionCommand.getTicketIdRegex();
+		const ticketMatches: Set<string> = new Set();
+		while ( ( ticketMatch = regex.exec( content ) ) !== null ) {
+			ticketMatches.add( ticketMatch[1] );
+		}
+		return Array.from( ticketMatches );
```

Comment:
There's no reason to use a set here.
```suggestion
		const ticketMatches: string[] = [];
		while ( ( ticketMatch = regex.exec( content ) ) !== null ) {
			ticketMatches.push( ticketMatch[1] );
		}
		return ticketMatches;
```